.github/tests/nested/values.yaml

```diff
@@ -0,0 +1,16 @@
+spire-server:
+  enabled: true
+  upstreamAuthority:
+    spire:
+      enabled: true
+      server:
+        address: spire-server.nested-root-server
+  controllerManager:
+    enabled: true
+    identities:
+      spiffeIDTemplate: spiffe://{{ .TrustDomain }}/k8s/{{ .ClusterName }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}
```

Comment:
This might be the default if I'm not mistaking 🤔

Reply:
The default does not have .ClusterName in it. Otherwise the same.

We might consider changing the default to .ClusterName to make it easier from going from basic to federation/tested later smoother?

Reply:
Through chatting with various folks and looking at some other projects defaults, I think this template is still probably best for nested and not for other cases.